tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Install the runnable products so that Xcode can run it. This includes `.app`s,
+# `.xctest`s, and also command line binaries.
+
+set -eux
+
+# Delete all logfiles that are older than 7 days
+find "${BAZEL_DIAGNOSTICS_DIR}" -type f -atime +7d -delete
+
+case "${PRODUCT_TYPE}" in
+    com.apple.product-type.framework)
+        input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}/${FULL_PRODUCT_NAME}")
+        ;;
+    com.apple.product-type.framework.static)
+	    # For static library, as the output is not under bazel-bin, we have to get it based on build event
+        # Example of such entry inside build event:
+        # important_output {
+        #   name: ".../SomeFramework.framework/SomeFramework"
+        #   uri: "file:///private/var/tmp/.../.../SomeFramework.framework/SomeFramework"
+        #   path_prefix:... 
+        #   ...
+        #  } 
+        # We only care about the entry ending with TARGET_NAME" so that we can get the path to its directory
+        QUERY=`grep -A 2 important_output "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep ${TARGET_NAME}\" | sed "s/uri: \"file:\/\///"`
```

Comment:
`grep ${TARGET_NAME}\"` is what I have concern with, because right now in order to get the right path that is single and deterministic, I am only looking for the one path ending with the binary file (which is usually just the target name, but it can be different if user sets it to be a diff. thing?)